src/PROPOSAL/PROPOSAL/math/Spherical3D.h

```diff
@@ -34,4 +34,22 @@ namespace PROPOSAL  {
     protected:
         void print(std::ostream&) const override;
      };
+
+
+    /*!
+     * Container for a spherical vector with radius 1
+     *
+     * \param zenith
+     * \param azimuth
+     */
+     struct UnitSphericalVector{
+        double zenith;
+        double azimuth;
+    
+        constexpr UnitSphericalVector(const UnitSphericalVector&) = default;
```

Comment:
This is already the implicitly-declared copy constructor and we can spare this line.
I think it's no good idea to explicit write it down because it's seems that it is necessary for each class.

Reply:
right, sorry, I forgot that the copy constructor is not removed when you define alternative constructors.

Reply:
But the default constructor has to be implemented back.